server/index.js

```diff
@@ -85,3 +102,12 @@ app.listen(process.env.PORT, () => {
   // eslint-disable-next-line no-console
   console.log(`express server listening on port ${process.env.PORT}`);
 });
+
+/*
+select *
+    from "workouts"
+    join "exerciseSets" using ("workoutId")
+    join "exercises" using ("exerciseId")
+    join "sets" using ("setId")
+   where "userId" = $1
+   */
```

Comment:
# ♻️

Reply:
😂  I save stuff all the way at the bottom in comments then forget about it, whoops!!